src/crsAdapter/TomaSPCAdapter.js

```diff
@@ -344,16 +349,43 @@ class TomaSPCAdapter {
     /**
      * @private
      * @param crsService object
-     * @returns {object}
+     * @param crsObject object
+     * @returns {{roomCode: *, mealCode: *, roomQuantity: (*|string|string), roomOccupancy: (*|string|string|string), children, destination: *, dateFrom: string, dateTo: string, type: string}}
      */
-    mapHotelServiceFromCrsObjectToAdapterObject(crsService) {
+    mapHotelServiceFromCrsObjectToAdapterObject(crsService, crsObject) {
+        const collectChildren = () => {
+            let children = [];
+            let travellerAssociation = crsService.travellerAssociation || '';
+
+            let startLineNumber = parseInt(travellerAssociation.substr(0, 1), 10);
+            let endLineNumber = parseInt(travellerAssociation.substr(-1), 10);
+
+            if (!startLineNumber) return;
+
+            do {
```

Comment:
I'm not sure yet how these files are invoked. Could we factor out some common code like this into a helper class?

Reply:
I would love to do that - but we should have a clear idea before of how the code structure should look like.

Let us postpone it to a point after the MVP is released.